src/components/formly.field.ts

```diff
@@ -66,6 +66,7 @@ export class FormlyField implements DoCheck, OnInit {
 
   ngOnInit() {
     this.createChildFields();
+    this.checkDefaultValue();
```

Comment:
why we need to check DefaultValue here, checking in [FormlyForm::addFormControl](https://github.com/formly-js/ng2-formly/blob/master/src/components/formly.form.ts#L110) is enough I think!